memote/suite/tests/test_sbo.py

```diff
@@ -261,8 +263,11 @@ def test_sink_specific_sbo_presence(read_only_model):
     ann = test_sink_specific_sbo_presence.annotation
     ann["data"] = get_ids(sbo.check_component_for_specific_sbo_term(
         helpers.find_sink_reactions(read_only_model), "SBO:0000632"))
-    ann["metric"] = len(ann["data"]) / len(
-        helpers.find_sink_reactions(read_only_model))
+    try:
+        ann["metric"] = len(ann["data"]) / len(
+            helpers.find_sink_reactions(read_only_model))
+    except ZeroDivisionError:
+        pytest.skip()
```

Comment:
What's missing here:
* Store a message `ann["message"] = "No sink reactions found.".
* Just in case set the metric to fully failed `ann["metric"] = 1.0.
* Set that message as the skip reason `pytest.skip(ann["message"])`.